frank/fit.py

```diff
@@ -21,6 +21,7 @@
    and output results. Alternatively a custom parameter file can be provided.
 """
 
+from frank import io, geometry, make_figs, radial_fitters, utilities
```

Comment:
Sorry can you change this back? Marco specifically asked that all frank imports go after standard python imports.

Reply:
Sure